src/components/tab/WorkspaceTab.vue

```diff
@@ -0,0 +1,93 @@
+
+<template>
+<el-card class="box-card">
+        <div slot="header" class="clearfix">
+            <slot name="title">Workspace</slot>
+        </div>
+        <slot name="items">
+            <el-tabs v-model="activeName" :title="title" :tab-position="tabPosition" @tab-click="$emit('click')">
+            <slot></slot>
+            </el-tabs>
+        </slot>
+    </el-card>
+</template>
+
+<script>
+// import { action } from '@storybook/addon-actions'
+
+export default {
+  name: 'WorkspaceTab',
+  props: {
+    title: {
+      type: String,
+      default: ''
+    },
+    tabPosition: {
+      type: String,
+      default: ''
+    },
+    activeName: {
+      type: String,
+      default: ''
+    }
+  },
+  methods: {
+    click (tab, event) {
+      console.log(tab, event)
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+@import "../../styles/theme";
+
+// tab styling
+.el-tabs_item {
+    font-family: $main-font-family;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 22px;
+    color: $dark-body-grey;
+    text-align: left;
+}
+
+.el-tabs__item.is-active {
+    color: $primary-color;
+    background: $primary-fill;
+    background-color: $primary-fill;
+    border-radius: 0px 40px 40px 0px;
+}
+
+.el-tabs__active-bar {
+  background-color: transparent;
+}
+
+.el-tabs {
+  text-align: left;
+}
+
+// card styling
+  .el-card {
```

Comment:
Found this other class changes the shadow as well: `.el-card.is-always-shadow`

I played around with the shadow to dim it down a bit for it to look like the wireframe. You can check it out :)
`box-shadow: inset 0px 3px 3px rgba(50, 50, 50, 0.25);`